bin/upgrading/update-db.pl

```diff
@@ -701,47 +701,6 @@ sub populate_moods {
     }
 }
 
-sub populate_mogile_conf {
```

Comment:
Here you remove the definition of populate_mogile_conf but not the place earlier in the file where the function is called.